### PR TITLE
[ADD] CRM: Merge Leads and Opps

### PR DESCRIPTION
**Pull request by jero-odoo**
Adding new CRM document explaining the process of identifying and merging similar leads and opportunities. (There is currently a section on the "Converting leads and opps document" that briefly touches on this, but it can be a complicated process and deserves its own doc)

### CODE REVIEW

**Comment by jero-odoo** [comment]
Hey @ksc-odoo this is ready for review. Thanks!

File: content/applications/sales/crm/pipeline/merge_similar.rst [hunk 1]
```diff
@@ -0,0 +1,113 @@
...
+After confirming that the leads/opportunities should be merged, return to the Kanban view using
+breadcrumbs, or by clicking the :guilabel:`Similar Lead` smart button. Click the :guilabel:`≣ (view
+list)` icon to change to list view.
+
+Check the box on the left of the page for the leads/opportunities to be merged. Then, click the
+:guilabel:`⚙️ Actions (gear)` icon at the top of the page, to reveal a drop-down menu. From that
+drop-down menu, select the :guilabel:`Merge` option to merge the selected opportunities or leads.
+
+When :guilabel:`Merge` is selected from the :guilabel:`⚙️ Actions (gear)` drop-down menu, a
+:guilabel:`Merge` pop-up modal appears. In that pop-up modal, under the :guilabel:`Assign
```
Comment (ksc-odoo) [inline content/applications/sales/crm/pipeline/merge_similar.rst:52]: when it comes to "pop-up modal" vs. "pop-up window/pop-up form" -- we've actually been having a discussion about this recently. While "modal" is technically accurate, and how it shows up in the 'Developer Tools' -- I'm so sure that the average user/reader is aware of such a term, and is likely more accustomed to seeing "window/form." We haven't landed on anything *concrete* as of yet, so if you feel most comfortable with "modal," feel free to keep using it. But, if you're aiming to deliver a more approachable terminology, my opinion/suggestion would be to go with a more user-friendly "pop-up window/form" route. But, again, totally your call -- just figured I'd throw it out there for you to consider.

File: content/applications/sales/crm/pipeline/merge_similar.rst [hunk 2]
```diff
@@ -0,0 +1,113 @@
...
+   :alt: List of similar leads and opportunities selected for merge in the CRM app.
+
+.. danger::
+   Merging is an irreversible action. Do **not** merge leads/opportunities unless absolutely certain
+   they should be combined.
+
+When leads and opportunities should not be merged
+=================================================
+
+There may be instances where a similar lead or opportunity is identified, but should not be merged.
```
Comment (ksc-odoo) [inline content/applications/sales/crm/pipeline/merge_similar.rst:70]:
```suggestion
There may be instances where a similar lead or opportunity is identified, but should *not* be merged.
```

**Review by ksc-odoo — APPROVED** [review]
hey @jero-odoo -- just finished my Final Review of this PR! Great work expanding on this very important CRM topic. I agree with you -- this *did* deserve its own doc. Nice job! I only have a handful of suggestions/comments that require your attention. All of them are fairly minor, so I'll approve this now. Once you've addressed the feedback, and implemented the necessary adjustments, feel free to tag this for Tech Review. Thanks! 👍

File: content/applications/sales/crm/pipeline/merge_similar.rst [hunk 3]
```diff
@@ -0,0 +1,113 @@
...
+Odoo automatically detects similar *leads* and *opportunities* within the *CRM* app. Identifying
+these duplicated records allows them to be merged without losing any information in the process.
+Not only does this help keep the *pipeline* organized, but it also prevents customers from being
+contacted by more than one salesperson.
+
+.. note::
+   When merging opportunities, no information is lost. Data from the other opportunity is logged in
+   the chatter, and the information fields, for reference.
+
+Identifying similar leads and opportunities
```
[the remark this answers is not in the document]
Comment (jero-odoo) [reply content/applications/sales/crm/pipeline/merge_similar.rst:14]: You are right, I think "Identify" instead of "Identifying" works

**Comment by jero-odoo** [comment]
Hey @samueljlieber this is ready for tech review. Thanks!

File: content/applications/sales/crm/pipeline/merge_similar.rst [hunk 4]
```diff
@@ -0,0 +1,115 @@
...
+   opportunity are being merged, the resulting record is referred to as an opportunity, regardless
+   of which record was created first.
+
+After confirming that the leads/opportunities should be merged, return to the Kanban view using the
+breadcrumb link, or by clicking the :guilabel:`Similar Leads` smart button. Click the :guilabel:`≣
+(view list)` icon to change to list view.
+
+Check the box on the left of the page for the leads/opportunities to be merged. Then, click the
+:guilabel:`⚙️ Actions (gear)` icon at the top of the page, to reveal a drop-down menu. From that
+drop-down menu, select the :guilabel:`Merge` option to merge the selected opportunities or leads.
```
Comment (samueljlieber) [inline content/applications/sales/crm/pipeline/merge_similar.rst:48]: Optional, no need to include an icon descriptor when the icon is already accompanied by a label "Actions"
```suggestion
:guilabel:`⚙️ Actions` icon at the top of the page, to reveal a drop-down menu. From that drop-down
menu, select the :guilabel:`Merge` option to merge the selected opportunities or leads.
```

File: content/applications/sales/crm/pipeline/merge_similar.rst [hunk 5]
```diff
@@ -0,0 +1,115 @@
...
+
+When :guilabel:`Merge` is selected from the :guilabel:`⚙️ Actions (gear)` drop-down menu, a
+:guilabel:`Merge` pop-up modal appears. In that pop-up modal, under the :guilabel:`Assign
```
Comment (samueljlieber) [inline content/applications/sales/crm/pipeline/merge_similar.rst:51]:
```suggestion
When :guilabel:`Merge` is selected from the :guilabel:`⚙️ Actions` drop-down menu, a
```

File: content/applications/sales/crm/pipeline/merge_similar/select-merge.png
Comment (samueljlieber) [inline content/applications/sales/crm/pipeline/merge_similar/select-merge.png:1]: This image is too wide at 984px, please try to keep images below 933px wide :)

**Review by samueljlieber — APPROVED** [review]
Hi @jero-odoo, awesome job on this brand new CRM doc! I have a few very small corrections, please see below. I am approving now and delegating merge to you 👍
.....
@robodoo delegate=jero-odoo

**Comment by jero-odoo** [comment]
@robodoo r+

**Pull request closed without merging**